app/models/case_contact.rb

```diff
@@ -38,8 +42,10 @@ class CaseContact < ApplicationRecord
     where(want_driving_reimbursement: want_driving_reimbursement) if want_driving_reimbursement == true || want_driving_reimbursement == false
   }
   scope :contact_type, ->(contact_type = nil) {
-    where("? = ANY (contact_types)", contact_type) if contact_type.present?
+    joins(:db_contact_types)
+      .where("contact_types.name in (?)", contact_type) if contact_type.present?
```

Comment:
Certainly not an issue, but for consistency with how this has been done in other places:

```suggestion
      .where(contact_types: { name: contact_type }) if contact_type.present?
```